pom.xml

```diff
@@ -589,6 +600,22 @@
         <!--/Tests-->
     </dependencies>
 
+    <!-- TODO: can be removed when upgraded in org.mock-server.mockserver-netty -->
```

Comment:
Let's ellaborate more on this comment, so that it's easier to understand when this section can be removed.